redis.tf

```diff
@@ -1,26 +1,27 @@
-resource "aws_elasticache_cluster" "redis" {
-  count                = var.redis_node_count > 0 ? 1 : 0
-  cluster_id           = "${var.project_environment}"
-  engine               = "redis"
-  apply_immediately    = var.redis_apply_immediately
-  subnet_group_name    = aws_elasticache_subnet_group.redis_subnet[0].name
-  node_type            = var.redis_node_type
-  num_cache_nodes      = var.redis_node_count
-  engine_version       = var.redis_engine_version
-  parameter_group_name = var.redis_parameter_group_name
-
-  port                 = 6379
-  security_group_ids   = [aws_security_group.production-redis[0].id]
+resource "aws_elasticache_replication_group" "redis" {
+  replication_group_id          = "${var.project_environment}"
+  description                   = "${var.project_environment} valkey cluster"
+  count                         = var.redis_cluster_count > 0 ? 1 : 0
```

Comment:
I'm sure you copied this over, but why is this not just set to `var.redis_cluster_count`?

Reply:
Oh nvm I see how it's used down below, this makes sense.